src/clj/rems/workflow/dynamic.clj

```diff
@@ -407,17 +494,35 @@
       (testing "submitted"
         (is (= #{::add-member}
                (possible-commands "applicant" submitted)))
-        (is (= #{::approve ::reject ::return ::request-decision}
+        (is (= #{::approve ::reject ::return ::request-decision ::request-comment}
                (possible-commands "assistant" submitted)))
         (is (= #{}
                (possible-commands "somebody else" submitted))))
+      (let [requested (apply-events submitted [{:event :event/comment-requested
+                                                :actor "assistant"
+                                                :commenter "commenter"}])]
+        (testing "comment requested"
+          (is (= #{::add-member}
+                 (possible-commands "applicant" requested)))
+          (is (= #{::approve ::reject ::return ::request-decision ::request-comment}
+                 (possible-commands "assistant" requested)))
+          (is (= #{::comment}
+                 (possible-commands "commenter" requested))))
+        (let [commented (apply-events requested [{:event :event/commented
+                                                  :commenter "commenter"
+                                                  :comment "..."}])]
+          (testing "comment given"
+            (is (= #{::approve ::reject ::return ::request-decision ::request-comment}
+                   (possible-commands "assistant" requested)))
+            (is (= #{::comment}
+                   (possible-commands "commenter" requested))))))
```

Comment:
Is this really so? We also have a test "cannot comment twice".

Looks like these tests are using `requested` instead of `commented`.